### PR TITLE
Extra field in transaction structure

### PR DESCRIPTION
**Pull request by miiu96**
Added field `options` in transaction structure

### CODE REVIEW

**Review by bogdan-rosianu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]
PR is fine, but don't merge until the elrond-go branch is merged

**Comment by andreibancioiu** [comment]
> PR is fine, but don't merge until the elrond-go branch is merged

Let's merge it though - two-ways compatible, right? Or not?

**Review by andreibancioiu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**
